requirements.tox.txt

```diff
@@ -2,4 +2,5 @@ pluggy==0.7.1
 py==1.5.4
 six==1.11.0
 tox==3.2.1
-virtualenv==16.0.0
\ No newline at end of file
+typing==3.6.6; python_version < '3.5'
```

Comment:
If you can see build server history, there were stranbe error after commit abd5c1f5491815ebebca6bc95e4ca8ff5dc8191c, which was solved by this. I think the error was during package installing rather than in the runtime itself.

Reply:
This failure https://travis-ci.org/ebroecker/canmatrix/jobs/469955956 occured after I only added typing=3.6.6 into requirements. And disappeared with the condition `< 3.5`
Don't ask me why :-)

Reply:
The claim is that we shouldn't need this, but oh well.

> NOTE: in Python 3.5 and later, the typing module lives in the stdlib, and installing this package has NO EFFECT.